app/models/planet.py

```diff
@@ -0,0 +1,21 @@
+class Planet:
+    def __init__(self, id, name, description, signs_of_life=None):
+        self.id = id
+        self.name = name
+        self.description = description
+        self.signs_of_life = signs_of_life
+    
+    def get_dict(self):
```

Comment:
👍 Nice job incorporating this refactor from the live code. Personally, I prefer the name we showed (`to_dict`) to emphasize this is a conversion rather than a more general "getter" (getters a family of functions meant to read out individual attributes from a class type used in many languages). When I read a function name like `get_dict`, it makes me think there is an attrbiute in the class called `dict` that's being gotten, but that's not what's happening here. Maybe `get_as_dict` would give a clearer idea of the intent, but then again, I still prefer `to_dict` (we'll eventually make a parallel `from_dict`).